src/main/kotlin/dev/deftu/omnicore/api/OmniCore.kt

```diff
@@ -0,0 +1,71 @@
+@file:JvmName("OmniCore")
+
+package dev.deftu.omnicore.api
+
+import dev.deftu.eventbus.EventBus
+import dev.deftu.eventbus.bus
+import dev.deftu.eventbus.invokers.LMFInvoker
+
+//#if MC >= 1.16.5
+import net.minecraft.SharedConstants
+//#elseif FORGE
+//$$ import net.minecraftforge.common.ForgeVersion
+//#elseif FABRIC
+//$$ import net.fabricmc.loader.api.FabricLoader
+//#endif
+
+private val VERSION_REGEX = "(?<major>\\d+)\\.(?<minor>\\d+)(?:\\.(?<patch>\\d+))?".toRegex()
+
+public const val ID: String = "@MOD_ID@"
+public const val VERSION: String = "@MOD_VERSION@"
+public const val GIT_BRANCH: String = "@GIT_BRANCH@"
+public const val GIT_COMMIT: String = "@GIT_COMMIT@"
+public const val GIT_URL: String = "@GIT_URL@"
+
+public var minecraftVersion: String = "unknown"
+    private set
+    get() {
+        if (field != "unknown") {
```

Comment:
can you cache these values via `lazy`? They wont change so like

Reply:
Maybe! I'll take a look, but it's already cached as-is.